packages/numpy/patches/fix-install-with-skip-build.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/numpy/distutils/command/install_clib.py b/numpy/distutils/command/install_clib.py
+index 662aa00bd..6f51e22ca 100644
+--- a/numpy/distutils/command/install_clib.py
++++ b/numpy/distutils/command/install_clib.py
+@@ -20,6 +20,9 @@ class install_clib(Command):
+     def run (self):
+         build_clib_cmd = get_cmd("build_clib")
+         build_dir = build_clib_cmd.build_clib
++        if build_dir is None:
++            build_clib_cmd.finalize_options()
++            build_dir = build_clib_cmd.build_clib
```

Comment:
Numpy includes its own set of monkey patches to distutils, which includes a bug that breaks `setup.py install --skip-build`.  This fixes that.